network.tf

```diff
@@ -1 +1,62 @@
+provider "google" {
+  project = var.project_id
+}
 
+module "gcp-network" {
```

Comment:
Is there a way to add tags to all created resources.
Use `data.ns_workspace.this.tags` to add a standard set of tags.

Reply:
GCP has "Label" concept similar to the AWS resource tag however, it's limited to very few services and the VPC network and Subnet are not part of that. [GCP component with Label support](https://cloud.google.com/compute/docs/labeling-resources#using-labels)

Also, GCP has [Network tag](https://cloud.google.com/compute/docs/vpc/add-remove-network-tags)  but it is used for a different purpose.

Reply:
That's unfortunate although GCP having projects to isolate resources is much nicer.
Feel free to leave how it is.